docs/contributing-guide/6. submitting-a-pr.md

```diff
@@ -50,22 +52,22 @@ rush changelog
 
 This will start an interactive tool that will:
 
-1. First ask you to select whether your change is a Major, Minor, or Patch (for more information on which to choose see: [semver docs](https://semver.org/))
+1. First ask you to select whether your change is a Major, Minor, or Patch (for more information on which to choose see [Semantic Versioning](https://semver.org/)).
 
 1. Then ask for a short description of your change - this is what will be included in the changelog when a new version is released, so keep it concise and meaningful. See [Tips for writing meaningful changelog entries](../reviewer-notes/tips-for-writing-changelog-entries).
 
-### Generating new snapshot for your UI changes
+### Generating new snapshots for your UI changes
 
-If you are making an ui change in your PR, you might invalidate some screen snapshot in our automation visual tests.
+If you are making a UI change in your PR, you might invalidate some screen snapshots in our automated visual tests.
 
-1. Check the failing github actions to see if that is a desired UI change
+1. Check the failing Github Actions to see if the failure is due to a UI change.
```

Comment:
```suggestion
1. Check the failing Github Actions to see if the failure was caused by a UI change.
```

Reply:
It could have better consistency here by using `is` because we are keep using `present tense` in this paragraph